Master

yxc775 opened the pull request:
playing around

jlbenskin commented:
Conflicting files
20160819_OlympicSportsByCountries.csv 
20170816_Documenting_Hate.csv

jlbenskin reviewed (commented):
Commit looks perfect for merge.

jlbenskin commented:
Conflicts resolved.

leejenny1 committed:
covid update

daily update

leejenny1 committed:
covid daily update

update

leejenny1 committed:
covid update

daily update

leejenny1 committed:
US update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid upate

update

leejenny1 committed:
covid states

Update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid states

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid update

update

leejenny1 committed:
covid

update

leejenny1 committed:
upload

covid

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

Update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
issues

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
issues

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
issues

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

Update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
issues

update

leejenny1 committed:
covid

update

leejenny1 committed:
issues

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

updates

leejenny1 committed:
issues

Update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leejenny1 committed:
covid

update

leslie-leuenberger committed:
US_weeklyPoliticalIssues

updated file

annafreeland committed:
Create 20200922_nz_party_issues.csv

annafreeland committed:
Revert "Create 20200922_nz_party_issues.csv"

This reverts commit eb8e0a3aad4c5c9108cf01d2284a677696d6f23d.

annafreeland committed:
Election Issues searched with New Zealand Political Parties

Percentage share of search interest for key election issues as searched alongside political parties in New Zealand. Covers week leading to 22 September 2020.

Commit:
US Political Issues Update

Updated US political issues with weeks beginning 13th and 20th Sep.

keilaguimaraes committed:
Halloween Costumes

Top searched Halloween costumes, US, September 2020

Commit:
US political issues update

New weekly data added

leejenny1 committed:
issues update

updated

yippeyay committed:
Update political trends

keilaguimaraes committed:
Interesse pelos partidos nas capitais

Distribuição de interesse de busca, nas capitais brasileiras, pelos 10 partidos mais buscados entre 15.11 e 16.11

keilaguimaraes committed:
Interesse pelos partidos nas capitais

keilaguimaraes committed:
Merge branch 'master' of https://github.com/GoogleTrends/data

keilaguimaraes committed:
Interesse por partidos nas capitais

keilaguimaraes committed:
Share of search for political parties across capital cities in Brazil, between 15.11 and 16.11

keilaguimaraes committed:
Share of search for political parties across capital cities in Brazil, between 15.11 and 16.11

yippeyay committed:
Add files via upload

keilaguimaraes committed:
Distribuição no interesse de busca

Nos candidatos das capitais que terão segundo turno em 2020

keilaguimaraes committed:
Merge branch 'master' of https://github.com/GoogleTrends/data

keilaguimaraes committed:
20201123_Distribuição_interesse_candidatos_capitais

yippeyay committed:
Add files via upload

yippeyay committed:
Delete US_weeklyPoliticalIssues.xlsx

yippeyay committed:
Delete US_weeklyPoliticalIssues (1).xlsx

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Covid Weekly Update

Covid Weekly Update

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Create 12082020_Coronavirus_WeeklyTrendsdata.xlsx

leslie-leuenberger committed:
Update 12082020_Coronavirus_WeeklyTrendsdata.xlsx

leslie-leuenberger committed:
Delete 12082020_Coronavirus_WeeklyTrendsdata.xlsx

leslie-leuenberger committed:
Create 12082020_Coronavirus_WeeklyTrendsData.xlsx

leslie-leuenberger committed:
20201214_Coronavirus_WeeklyTrendsdata

20201214_Coronavirus_WeeklyTrendsdata

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Create 20210106_Coronavirus_WeeklyTrendsdata.xlsx

leslie-leuenberger committed:
Create 20210111_Coronavirus_WeeklyTrendsdata.xlsx

leslie-leuenberger committed:
Weekly Covid Data, 2021

Weekly Covid Data, 2021

leslie-leuenberger committed:
Weekly Covid Data, 2021

Weekly Covid Data, 2021

leslie-leuenberger committed:
delete 2021011 covid data

delete covid weekly file

leslie-leuenberger committed:
Covid weekly data 20210111

covid weekly data 20210111

leslie-leuenberger committed:
delete file

delete covid weekly file

leslie-leuenberger committed:
create new file covid weekly update

 new covid weekly update

leslie-leuenberger committed:
Update 20210111_Coronavirus_WeeklyTrendsdata.xlsx

covid weekly update

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_weekly_20210118

Coronavirus_weekly_20210118 Update

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
coronavirus weekly update

adding coronavirus file and deleting old coronavirus weekly update

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus weekly data 02/01/2021

Coronavirus weekly data update 02/01/2021

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
covid weekly data update, 02102021

covid weekly data update, 02102021

leslie-leuenberger committed:
Coronavirus Weekly Update 02/15/2021

Coronavirus Weekly Update 02/15/2021

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Covid Weekly Update 02232021

Covid Weekly Update 02232021

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus Weekly Data 8.3.2021

Coronavirus Weekly Data 8.3.2021

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus Update 03172021

Coronavirus Update 03172021

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
coronavirus weekly update 20210329

coronavirus weekly update 20210329

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_04072021

Coronavirus_WeeklyTrendsdata_04072021

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Covid Weekly Update

Coronavirus_WeeklyTrendsdata_20210414

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Removing and adding Covid Weekly Updates

Removing old Corona Weekly datasets and adding a new one for 23.4.2021 (previous week)

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Covid Weekly Update

weekly update 572021

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Weekly Coronavirus Update 17.5.2021

Weekly Coronavirus Update 17.5.2021

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Deleting old covid updates and adding new covid file

Deleting old covid updates and adding new covid file

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Covid Weekly Update

Weekly Covid data update (23.6.2021)

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_07122021

Coronavirus_WeeklyTrendsdata_07122021

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
add new files and delete old ones

Coronavirus_WeeklyTrendsdata_20210718
add new files and delete old ones

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_20210726

Coronavirus_WeeklyTrendsdata_20210726

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_20210819

remove old files and add Coronavirus_WeeklyTrendsdata_20210819

jckr committed:
replacing tabletop.js with papa.parse to address the deprecation of google sheets api v3

SimonRogers committed:
Merge pull request #34 from GoogleTrends/papa-parse

replacing tabletop.js with papa.parse

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_09012021

Coronavirus_WeeklyTrendsdata_09012021

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_2021_09_19

Coronavirus_WeeklyTrendsdata_2021_09_19

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Update .DS_Store

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_20210928

Coronavirus_WeeklyTrendsdata_20210928

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_2021_10_29

Coronavirus_WeeklyTrendsdata_2021_10_29

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

yippeyay committed:
Add files via upload

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_01052022

Coronavirus_WeeklyTrendsdata_01052022

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_20220218

Coronavirus_WeeklyTrendsdata_20220218

leslie-leuenberger committed:
Coronavirus_WeeklyTrendsdata_03152022

Coronavirus_WeeklyTrendsdata_03152022

SimonRogers committed:
Update index.html

SimonRogers committed:
Update index.html

gabrielacaesar committed:
adding project elections in search (it also has its own repo)

https://github.com/GoogleTrends/elections-in-search

gabrielacaesar committed:
folder test

gabrielacaesar committed:
newsletter 11 outubro

alyssafrndz committed:
Beatles songs ranked since 2004

Beatles songs ranked since 2004, worldwide. Including US and UK percent change from 2021 to 2022

alyssafrndz committed:
Beatles Songs Ranked Since 2004, Worldwide

Beatles songs ranked since 2004, worldwide.

leslie-leuenberger committed:
Update .DS_Store

leslie-leuenberger committed:
Energy Trends weekly

Energy Trends weekly

gabrielacaesar committed:
newsletter #10 br election

gabrielacaesar committed:
jenny's files

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
folder israel

gabrielacaesar committed:
israel election photos

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
newsletter election #11

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
#14 edition

gabrielacaesar committed:
#15

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
BR election newsletter #16 edition

leslie-leuenberger committed:
World Cup 2022_ Search interest for the German football players (in german)

World Cup 2022_ Search interest for the German football players (in german)

kseaton327 committed:
CA Halloween costumes data set

Commit:
Create 2022-11-21 World Cup teams by country past week.csv

gabrielacaesar committed:
economy, thirteenth salary - data and questions

leslie-leuenberger committed:
World Cup 2022_ Worldwide search interest for all 32 teams by day

World Cup 2022_ Worldwide search interest for all 32 teams by day

MisatoNakayama committed:
Add files via upload

Thanksgiving and Christmas trending recipes in the US in November 2022.

MisatoNakayama committed:
Add files via upload

Trending recipes over the past year and Thanksgiving and Christmas recipes over the past month in New York City.

MisatoNakayama committed:
Delete 20221110_HolidayRecipe_US.csv - Trending recipes.csv

MisatoNakayama committed:
Delete 20221110_HolidayRecipe_NYC.csv - Trending recipe in NYC.csv

MisatoNakayama committed:
Add files via upload

Trending recipes over the past year and top Thanksgiving and Christmas recipes over the past month in the US.

MisatoNakayama committed:
Add files via upload

Trending recipes over the past year and top Thanksgiving and Christmas recipes over the past month in New York City.

kseaton327 committed:
Add files via upload

Top 10 most searched products in the US over Black Friday/Cyber Monday by category

gabrielacaesar committed:
easter - br

gabrielacaesar committed:
music festival - BR

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
cropped pics

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

kseaton327 committed:
Add files via upload

Top searched celebrities dataset: FR, DE, IN, IT, JP, MX, ES, TW, UK, US

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Delete amazon.png

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

keilaguimaraes committed:
Environmental issues per country, 2012-2023

Search interest in environmental issues and natural disasters, per country, from 2012 to 2023. Data for 2023 covers Jan-Mar

keilaguimaraes committed:
Environmental issues per country, 2012-2023

Search interest in environmental issues and natural disasters, per country, from 2012 to 2023. Data for 2023 covers Jan-Mar

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

gabrielacaesar committed:
Add files via upload

leslie-leuenberger committed:
Search interest for nuclear energy

Nuclear energy

kseaton327 committed:
Add files via upload

US search interest in extreme weather types by year

kseaton327 committed:
Top Drag Race contestants

Top 25 RuPaul's Drag Race contestants, ranked by US Google Search interest

gabrielacaesar committed:
new pic - zanin

pallavisingha1 committed:
Add files via upload

Weekly search interest in top searched candidates and parties, July to October 2023

pallavisingha1 committed:
Top searched Halloween costumes, Australia 2023

gabrielacaesar committed:
new logo to replace index.html picture

gabrielacaesar committed:
Update index.html

gabrielacaesar committed:
update logo

gabrielacaesar committed:
data pulls for mexico 09/2025
